source/Classroom/Basics/Security/security_checklist.md

```diff
@@ -1,49 +1,36 @@
 ---
 
-SendGrid doesn’t compromise when it comes to compromised accounts and neither should you. Every day that you put off security steps, your reputation as a business and a sender is at risk. Take a few minutes, go through this checklist and make sure that your systems are secure.
+SendGrid doesn’t compromise when it comes to compromised accounts and neither should you. Every day that you put off security steps, your reputation as a business and a sender is at risk. Take a few minutes and this checklist to help make ensure that your systems are secure.
 
-1)  Is your password robust?  (If your account is compromised, do a virus scan prior to changing your password in order to get rid of any key-logger's.)
+{% anchor h3 %} 1)  Use a robust password!  {% endanchor %}
 
-- Do you use the same password for multiple sites?
+- Don't use the same password for multiple sites, vary your passwords whenever possible.
+- If possible, use  upper and lower case letters, numerals, and special characters. 
+- Don’t use easily guessable words or phrases such as personal information, your company name, or abbreviations. 
+- Don't expose your password on Github!
 
-  - vary your passwords from site to site
+Consider setting up [Teammates]({{root_url}}/docs/User_Guide/Settings/teammates.html) to create, manage, and restrict additional sets of credentials to more precisily grant permissions within your SendGrid account.
 
-- Do you use both upper- and lower-case letters?
-- Do you use one or more numerical digits?
-- Do you use special characters? (e.g. @, #, $ etc.)
-- Don’t use easily guessable words or phrases such as:
+{% anchor h3 %} 2) Avoid having web browsers save your SendGrid password. {% endanchor %} 
 
-  - words found in a dictionary or the user's personal information.
-  - passwords that match the format of calendar dates, telephone numbers, or other common numbers.
-  - your company name or an abbreviation.
+{% anchor h3 %} 3) Perform regular virus scans on any computers that are used to send mail through SendGrid or are used log in to your account. {% endanchor %}
 
-2)  Do you avoid having web browsers save your password?  
+{% anchor h3 %} 4) Encrypt and secure wireless connections. {% endanchor %} 
 
-3)  Do you perform regular virus scans on any computers that are used to send mail through SendGrid or login to your account?  
+{% anchor h3 %} 5)  Avoid publicly displaying configuration files or web pages (Github, social media, etc.) that contain your SendGrid credentials. Make sure they are not indexable by search engines or visible in any way to outsiders. {% endanchor %} 
 
-4)  Are your wireless network connections secure and encrypted?  
+{% anchor h3 %} 6)  Are any of your web applications vulnerable?  Make sure these are up to date. (for example old versions of WordPress are vulnerable to exploitation).  Please review: https://codex.wordpress.org/Hardening_WordPress for hardening your security with WordPress. {% endanchor %} 
```

Comment:
~~Are any of your web applications vulnerable?  Make sure these are up to date. (for example old versions of WordPress are vulnerable to exploitation).~~ -> Check for vulnerabilities in your web applications, and always make sure they are completely up to date - often, software updates fix vulnerabilities. For example, old versions of WordPress are vulnerable to exploitation.  Review https://codex.wordpress.org/Hardening_WordPress to hardening your security with WordPress.